proto/worm.proto

```diff
@@ -1,14 +1,14 @@
 syntax = "proto3";
 
-package boundless.services;
+option go_package = "golang";
 
 import "Event.proto";
 import "Command.proto";
 
 // service definition.
 service WormService {
   // Issue a Command
-  rpc Send(Command) returns (Response) {}
+  rpc SendCommand(Command) returns (Response) {}
```

Comment:
i initially called this `Issue` because it's called on the service right? the service isnt sending a command, callers are giving a command to the service, so i thought having the service `Issue` a command made more sense.